core/src/main/scala/org/apache/spark/status/storeTypes.scala

```diff
@@ -129,7 +129,7 @@ private[spark] object TaskIndexNames {
   final val SER_TIME = "rst"
   final val SHUFFLE_LOCAL_BLOCKS = "slbl"
   final val SHUFFLE_READ_RECORDS = "srr"
-  final val SHUFFLE_READ_TIME = "srt"
+  final val SHUFFLE_READ_FETCH_WAIT_TIME = "srfwt"
```

Comment:
Wont this not make existing level db files break ?
If yes, we should revert the rename.

Reply:
In this scenario, we have not found the problem that the leveldb file cannot be parsed for the time being. But this seems to be a problem, we are glad to revert to this rename